extension.json

```diff
@@ -215,16 +210,17 @@
                   "type": "boolean"
                 }
               },
-              "required": ["namespace", "id", "authenticatedState"],
+              "required": ["id", "authenticatedState"],
               "additionalProperties": false
-            }
+            },
+            "minProperties": 1
           }
         },
-        "required": ["instanceName", "customerIds"],
+        "required": ["instanceName", "identities"],
         "additionalProperties": false
       },
-      "libPath": "dist/lib/actions/setCustomerIds/index.js",
-      "viewPath": "actions/setCustomerIds.html"
+      "libPath": "dist/lib/actions/syncIdentity/index.js",
+      "viewPath": "actions/syncIdentity.html"
```

Comment:
Previously, `customerIds` was an array and each customer ID object had a `namespace` property on it. When the Set Customer Ids action was run, we would transform the array into a object where keys were the namespaces and the values were the customer ID objects.

I presume the reason why `customerIds` in settings was an array is because that's how the customer IDs were managed within the view state (specifically, inside formik). However, I think it's most appropriate to not model the settings object according to view state, but instead what will be sent through Alloy, for a couple reasons:

1. It requires less transformation within code deployed to customer websites, since the data will already be in a map instead of an array.
2. While we can validate inside the extension view that there are no two identities with the same namespace, you can't validate that for Launch API users if we were to use the schema we previously had. If, however, our schema dictates that `identities` an object where the keys are the namespaces, however, the structure itself enforces that API users _can't_ get into a situation where they have multiple identities with the same namespace.